kokoro/gcp_ubuntu/kokoro_build.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+
+# Fail on any error.
+set -e
+
+termInstance() {
+  gcloud compute instances delete teradata-kokoro --zone us-central1-a --project="${GCP_PROJECT}" --quiet
+}
+
+#delete instance after error
+trap 'termInstance' ERR
+
+ls "${KOKORO_KEYSTORE_DIR}"
+#Variables
+export TD_PSW=$(<"${KOKORO_KEYSTORE_DIR}"/76474_teradata-12232021)
```

Comment:
Please enclose dynamic arguments in quotes to avoid interpretation issues due to unexpected spaces (I.e., `TD_PSW="$(...)"` in this case.

Reply:
Wrapped in double quotes